File: CS/WebSite/App_Code/DataClasses.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Data;

public class Invoice {
    private int id;

    public int Id {
        get { return id; }
        set { id = value; }
    }
    private string description;

    public string Description {
        get { return description; }
        set { description = value; }
    }
    private decimal price;

    public decimal Price {
        get { return price; }
        set { price = value; }
    }

    public Invoice(int id, string description, decimal price) {
        this.Id = id;
        this.Description = description;
        this.Price = price;
    }
}

public class ManualDataSet : DataSet {
    public ManualDataSet()
        : base() {
        DataTable table = new DataTable("table");

        DataSetName = "ManualDataSet";

        table.Columns.Add("ID", typeof(Int32));
        table.Columns.Add("MyDateTime", typeof(DateTime));
        table.Columns.Add("MyRow", typeof(string));
        table.Columns.Add("MyData", typeof(double));
        table.Constraints.Add("IDPK", table.Columns["ID"], true);

        Tables.AddRange(new DataTable[] { table });
    }

    public static ManualDataSet CreateData() {
        ManualDataSet ds = new ManualDataSet();
        DataTable table = ds.Tables["table"];

        table.Rows.Add(new object[] { 0, DateTime.Today, "A", 103 });
        table.Rows.Add(new object[] { 1, DateTime.Today, "B", 200 });
        table.Rows.Add(new object[] { 2, DateTime.Today, "C", 446 });

        return ds;
    }
}

File: VB/WebSite/App_Code/DataClasses.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web
Imports System.Data

Public Class Invoice
	Private id_Renamed As Integer

	Public Property Id() As Integer
		Get
			Return id_Renamed
		End Get
		Set(ByVal value As Integer)
			id_Renamed = value
		End Set
	End Property
	Private description_Renamed As String

	Public Property Description() As String
		Get
			Return description_Renamed
		End Get
		Set(ByVal value As String)
			description_Renamed = value
		End Set
	End Property
	Private price_Renamed As Decimal

	Public Property Price() As Decimal
		Get
			Return price_Renamed
		End Get
		Set(ByVal value As Decimal)
			price_Renamed = value
		End Set
	End Property

	Public Sub New(ByVal id As Integer, ByVal description As String, ByVal price As Decimal)
		Me.Id = id
		Me.Description = description
		Me.Price = price
	End Sub
End Class

Public Class ManualDataSet
	Inherits DataSet
	Public Sub New()
		MyBase.New()
		Dim table As New DataTable("table")

		DataSetName = "ManualDataSet"

		table.Columns.Add("ID", GetType(Int32))
		table.Columns.Add("MyDateTime", GetType(DateTime))
		table.Columns.Add("MyRow", GetType(String))
		table.Columns.Add("MyData", GetType(Double))
		table.Constraints.Add("IDPK", table.Columns("ID"), True)

		Tables.AddRange(New DataTable() { table })
	End Sub

	Public Shared Function CreateData() As ManualDataSet
		Dim ds As New ManualDataSet()
		Dim table As DataTable = ds.Tables("table")

		table.Rows.Add(New Object() { 0, DateTime.Today, "A", 103 })
		table.Rows.Add(New Object() { 1, DateTime.Today, "B", 200 })
		table.Rows.Add(New Object() { 2, DateTime.Today, "C", 446 })

		Return ds
	End Function
End Class
